webapp/store/content/distros/arch.yaml

```diff
@@ -2,6 +2,7 @@ name: Arch Linux
 color-1: "#1793D1"
 color-2: "#116F9E"
 logo: https://assets.ubuntu.com/v1/feca0fc0-Distro_Logo_ArchLinux.svg
+display-logo: true
```

Comment:
Can't we make it true by default? And only set to no in fedora?

Also 'display-logo' is a bit misleading - logo is still shown on a page, just not in background.